templates/trivy/trivy-sts.yaml

```diff
@@ -102,6 +102,10 @@ spec:
               value: {{ .Values.trivy.skipUpdate | default false | quote }}
             - name: "SCANNER_TRIVY_SKIP_JAVA_DB_UPDATE"
               value: {{ .Values.trivy.skipJavaDBUpdate | default false | quote }}
+            - name: "SCANNER_TRIVY_DB_REPOSITORY"
+              value: {{ .Values.trivy.dbRepository | join \",\" | quote }}
+            - name: "SCANNER_TRIVY_JAVA_DB_REPOSITORY"
```

Comment:
Hi @benji78 ,

I found the the `join \",\"` part would cause template error `Error: parse error at (harbor/templates/trivy/trivy-sts.yaml:106): unexpected "\\" in operand`
`join ","` without the escape would be sufficient

Reply:
<img width="1086" height="671" alt="Screenshot 2025-09-01 at 15 04 05" src="https://github.com/user-attachments/assets/2726c69d-f803-4761-a887-b169d2a9a6a5" />